Palindrome.UnitTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+// Added a test for Empty Input.
+// Created branch 'review' for PR sharing.
+
+
+namespace PalindromeArray.UnitTests
+{
+    [TestClass]
+    public class PalindromeUnitTests
+    {
+        [TestMethod]
+        public void Check_Empty_Input()
+        {
+            // Arrange
+            Palindrome check = new Palindrome();
+            char[] arrToCheck = check.PalindromeHelper("");
+            // Act
+            bool boolResultOfPalindromeCheck = check.IsArrayPalindrome(arrToCheck);
+            // Assert
+            Assert.AreEqual(false, boolResultOfPalindromeCheck);
+        }
+        [TestMethod]
+        public void Check_Longest_Palindrome_In_Dictionary()
+        {
+            Palindrome check = new Palindrome();
```

Comment:
This entire method is being repeated in every method. Why not make a helper method in this class:

```csharp
private void RunIsArrayPalindromeTest(string input, bool expectedOutput)
{
    Palindrome unit = new Palindrome();
    bool actualResult = unit.IsArrayPalindrome(input);
    Array.AreEqual(expectOutput, actualResult, $"{nameof(unit.IsArrayPalindrome)}(\"{input}\")");
}
```
You can then use the helper method to make your code easier to read.

Reply:
This is where I needed input and clarification on tests and helper methods. Will change.